src/components/create/create.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import 'jest-styled-components';
+import { shallow } from 'enzyme';
+import TestRenderer from 'react-test-renderer';
+import Create from './create.component';
+import classicTheme from '../../style/themes/classic';
+import CreateClassicStyle from './create-classic.style';
+import CreateStyle from './create.style';
+
+function render(props) {
+  return shallow(<Create { ...props }> Create component </Create>);
+}
+
+describe('Create', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = render({});
+  });
+
+  it('should render correctly', () => {
+    expect(TestRenderer.create(<CreateStyle />)).toMatchSnapshot();
+  });
+
+  it('should render correct', () => {
```

Comment:
Spelling on "correctly" in incorrect. I assume you've done that on purpose because of the two similar tests?

I don't think you need both of those snapshots, do you?

Reply:
The first one is for the <CreateStyle>, the second one is for <Create>

Reply:
I think you probably just need to update the `it`s to `should render Create / CreateStyle correctly`.